packages/react-components/src/theming/icons.tsx

```diff
@@ -131,7 +132,7 @@ const SitePermissionMic20Filled = (): JSX.Element => {
 };
 
 /* @conditional-compile-remove(call-readiness) */
-const SitePermissionCamera20Filled = (): JSX.Element => {
+const SitePermissionCamera20Filled = (): React.JSX.Element => {
```

Comment:
Noticed that all these have React prepended now.
Can we import in order not to have to call React.JSX.Element all the time rather just JSX.Element?

Reply:
This is a deprecation. We should be adopting this throughout.